projects/output-and-error-handling/client/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+)
+
+func main() {
+	res, err := http.Get("http://localhost:8080")
+	if err != nil {
+
+		if res != nil {
+			fmt.Println("Response status:", res.Status)
+			if res.StatusCode == http.StatusTooManyRequests {
+				fmt.Println("Server is too busy. Retry after:", res.Header.Get("Retry-After"))
```

Comment:
Can you make the code here do the retrying for you, instead of telling the user to do so themself?

Reply:
Hey Daniel @illicitonion 

Thanks for your reviewing my code, now I pushed my new change could you please review it